arcgis-ios-sdk-samples/Maps/Download preplanned map/DownloadPreplannedMapViewController.swift

```diff
@@ -20,7 +20,7 @@ import ArcGIS
 // MARK: - Constants
 
 private enum Constants {
-    static let activityIndicatorColor = UIColor.primaryBlue
+    static let activityIndicatorColor = UIColor.primaryThemeColor
```

Comment:
If we want the color of activity indicator's to match the global accent color, then we should do it through the appearance proxy, not in individual samples like this. That being said, I'm not sure that's what we want. The accent color is usually an indication of an interactive element, which an activity indicator is not. Because of that, I would suggest just removing this so that the default color is used.